openqa-investigate

```diff
@@ -51,7 +51,9 @@ clone() {
     base_prio=$(echo "$clone_job_data" | runjq -r '.job.priority') || return $?
     clone_settings=("TEST+=:investigate$name_suffix" '_TRIGGER_JOB_DONE_HOOK=1' '_GROUP_ID=0' 'BUILD=')
     if [[ $refspec ]]; then
+        testgiturl=$(echo "$clone_job_data" | runjq -r '.job.settings.TEST_GIT_URL') || return $?
```

Comment:
`TEST_GIT_URL` is only in `vars.json` though, it's not in the settings table

Reply:
So I assume @b10n1k you have not actually tested it? I suggest you call

```
env dry_run=1 openqa-investigate $job_url
```

with $job_url could be any openQA job but I suggest one that is actually not from os-autoinst-distri-opensuse, like https://openqa.opensuse.org/group_overview/24 using os-autoinst-distri-opensuse

After the "dry-run" you can also do a real run because actually the effect should be that *nly* up to 4 investigation jobs are triggered and a comment is written on the original job so not that harmful.

Reply:
note that you would have to find a test that failed plus there was a change in test code, to see the actual effect

Reply:
Here is one that you could use: https://openqa.opensuse.org/tests/3813400#comments

Reply:
Like discussed in chat, this is too old and it didn't have the TEST_GIT_URL feature yet.

Reply:
https://aquarius.suse.cz/tests/20051#settings  shows the correct CASEDIR after run the openqa-investigate

Reply:
https://github.com/os-autoinst/scripts/pull/278/files#r1453307820 is still not resolved. Can you comment on that?

Reply:
I'm wondering how that can possibly work, as the TEST_GIT_URL is not reported as part of the job.settings API response:
https://openqa.opensuse.org/tests/3875154/file/vars.json has it, but:
```
% openqa-cli api --o3 jobs/3875154 | jq .job.settings
{
  "ARCH": "x86_64",
  "BACKEND": "qemu",
  "BUILD": ":TW.25887",
  "DESKTOP": "minimalx",
  "DISTRI": "openqa",
  "FLAVOR": "dev",
  "HDDSIZEGB": "20",
  "HDD_1": "opensuse-Tumbleweed-x86_64-20240115-minimalx@64bit.qcow2",
  "MACHINE": "64bit-2G",
  "NAME": "03875154-openqa-Tumbleweed-dev-x86_64-Build:TW.25887-openqa_from_bootstrap@64bit-2G",
  "OPENQA_FROM_BOOTSTRAP": "1",
  "OPENQA_HOST": "http://openqa.opensuse.org",
  "QEMUCPU": "host",
  "QEMURAM": "2048",
  "TEST": "openqa_from_bootstrap",
  "VERSION": "Tumbleweed",
  "WORKER_CLASS": "qemu_x86_64"
}
```
So I would like to test this PR myself locally.

Reply:
Oh I can see it:
https://aquarius.suse.cz/tests/20047#settings has a TEST_GIT_URL in the  settings. How did it get there? Did you set it manually? It should only appear in the vars.json

Reply:
https://openqa.opensuse.org/tests/3875380 after PR update